operatorcourier/verified_manifest.py

```diff
@@ -9,6 +9,81 @@
 
 logger = logging.getLogger(__name__)
 FLAT_KEY = '__flat__'
+CRD_KEY = 'CustomResourceDefinition'
+CSV_KEY = 'ClusterServiceVersion'
+
+
+def is_manifest_folder(folder_path):
```

Comment:
See my comment above about how it seems like these helper functions don't belong in the same module as the verified_manifest class module.